doc/developer/workflow.rst

```diff
@@ -336,6 +336,11 @@ Pre-submission Checklist
   the new feature within our existing CI infrastructure.  Also the
   addition of automated testing to cover any pull request is encouraged.
 
+- All new features must use the current latest version of acceptable code.
+  As an example all new must use the typesafe list structures instead of the
```

Comment:
I guess "code" word is missing here.
As an example all new code must use...

Reply:
fixed